content/v2/domains.markdown

```diff
@@ -37,6 +37,20 @@ Responds with HTTP 200.
 <%= pretty_print_fixture("/listDomains/success.http") %>
 ~~~
 
+### Filters
+
+Name | Type | Description
+-----|------|------------
+`:name_like` | `wildcard` | Wildcard search for domain name
```

Comment:
I'd slightly change the description to be more understandable to whoever was not involved in the decision process. Something like:

> Only include domains containing given string.

I would also remove the "type" concept, as this is something somehow internal.